client/src/hooks/use-modal.ts

```
@@ -0,0 +1,296 @@
+ * modals.help.isOpen;
+ * ```
+ */
+export function useMultipleModals(
+  modalIds: string[],
+  config: ModalConfig = {}
+): Record<string, UseModalReturn> {
+  const modals: Record<string, UseModalReturn> = {};
+
+  modalIds.forEach(id => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    modals[id] = useModal({
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)